src/Illuminate/Pagination/LengthAwarePaginator.php

```diff
@@ -191,41 +185,12 @@ public function lastPage()
      */
     public function toArray()
     {
-        return [
-            'current_page' => $this->currentPage(),
-            'data' => $this->items->toArray(),
-            'first_page_url' => $this->url(1),
-            'from' => $this->firstItem(),
+        return array_merge(parent::toArray(), [
```

Comment:
Here it's enough just to add the custom data related to `LengtAwarePaginator` class, the common ones come from the parent.